java/maven.grammar/src/org/netbeans/modules/maven/codegen/NewMirrorPanel.java

```diff
@@ -135,18 +133,14 @@ public void changedUpdate(DocumentEvent e) {
         comUrl.setModel(urlmodel);
 
         btnLink.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
-        btnLink.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                try {
-                    URL link = new URL("http://maven.apache.org/guides/mini/guide-mirror-settings.html"); //NOI18N
-                    HtmlBrowser.URLDisplayer.getDefault().showURL(link);
-                } catch (MalformedURLException ex) {
-                    Exceptions.printStackTrace(ex);
-                }
+        btnLink.addActionListener(e -> {
+            try {
+                URL link = new URL("http://maven.apache.org/guides/mini/guide-mirror-settings.html"); //NOI18N
+                HtmlBrowser.URLDisplayer.getDefault().showURL(link);
+            } catch (MalformedURLException ex) {
+                Exceptions.printStackTrace(ex);
             }
         });
-        checkCentral();
```

Comment:
@sdedic to answer your question from the other PR, the NPE was here:
https://github.com/apache/netbeans/blob/a9845e3070f96255d000963ef7039e4a5b3165ad/java/maven.grammar/src/org/netbeans/modules/maven/codegen/NewMirrorPanel.java#L167

`nls` was null. This can happen when the mirror already exists and the user opens the dialog again, the check will run right from the constructor, `nls` is set post-construction and is therefore still null. By moving the check into `addNotify()` this is avoided.